src/main/kotlin/ar/edu/unahur/obj2/vendedores/Vendedor.kt

```diff
@@ -42,10 +46,18 @@ class Viajante(val provinciasHabilitadas: List<Provincia>) : Vendedor() {
   override fun puedeTrabajarEn(ciudad: Ciudad): Boolean {
     return provinciasHabilitadas.contains(ciudad.provincia)
   }
+  fun sumaProvHabilitadas() = provinciasHabilitadas.sumBy{ c -> c.poblacion}
```

Comment:
Bien por crear una subtarea, aunque no cuesta nada poner el nombre sin abreviaturas (que no son obvias). 
```suggestion
  fun sumaProvinciasHabilitadas() = provinciasHabilitadas.sumBy{ c -> c.poblacion}
```